Practica1/firstDeliverable.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+type data struct {
+	ID   int
+	Name string
+}
+
+func main() {
+	http.HandleFunc("/readCSVFile", handlerReadFile)
+	http.HandleFunc("/searchID", handlerSearchID)
+	fmt.Println("The server is running")
+
+	/*e := http.ListenAndServe(":8090", nil)
+
+	if e != nil {
+		log.Fatal(e)
+	}*/
+	go http.ListenAndServe(":8090", nil)
+
+	res, err := http.Get("http://127.0.0.1:8090/readCSVFile")
```

Comment:
You don;t need to call your handlers, they will be called when the users make a request